src/frontend/src/styles/main.css

```diff
@@ -250,7 +244,7 @@ main {
 }
 
 .hidden {
-  visibility: hidden;
```

Comment:
I suspect this will visually break the warning box in the recovery phrase; `display: none` removes the element from the flow, which means that when warnings show up, the page and buttons will move (not ideal). I'm happy to add a screenshot test for that actually

Reply:
Yeah, it also had other visual impacts on the device add flow -> reverted.